ibind/client/ibkr_client.py

```diff
@@ -62,11 +75,23 @@ def __init__(
         if url is None:
             url = f'https://{host}:{port}{base_route}'
 
+        
         self.account_id = account_id
+        self._use_oauth=use_oauth
         super().__init__(url=url, cacert=cacert, timeout=timeout, max_retries=max_retries)
+        
+        if self._use_oauth:
+            self.live_session_token,self.live_session_token_expires_ms=self.req_live_session_token()
```

Comment:
Good question. The live session token times out after 24hours, and I think it would need to be re-created if the session drops out. Maybe also allow the live session token to be re-created in a function?
It took some fiddling to get the order of the attributes correct (super before if self._use_oauth etc.), can you see any room for improvement?

Reply:
Well we do get that `live_session_token_expires_ms` value right - that should tell us when it expires? 

Or even better - we could observe what does IBKR return back when we try to call its endpoints with an expired live_session_token. Then we catch that error and request a new live session token when it happens. No need to keep track of the expiry date, just handle it once it happens. What do you recon?

Reply:
Yes, I could check what get returns after the expiry date, but I think there are other bigger bugs to fix right now.

Reply:
Sure. Can you add a TODO comment in here so that we remember to handle this later? We shouldn't release until we have a nice way to handle it

Reply:
Ok I'll add a comment.